maintaining/maintainers-guide.md

```diff
@@ -31,7 +23,7 @@ manage aspects of the open source project. These are:
 * **Release Process**: Manage Bazel's release process.
 * **Green Team**: Grow a healthy ecosystem of rules and tools.
 * **Developer Experience Gardeners**: Encourage external contributions, review issues
-  and pull requests, make our development workflow more open, and shift the team 
+  and pull requests, make our development workflow more open, and shift the team
```

Comment:
The "and shift the team towards github-first development" 
should be dropped. That is not an objective of that group.
It is more of an overall aspirational goal for the project.

Reply:
Took the wording out of the paragraph.